app/src/main/kotlin/de/digitalService/useID/ui/components/pin/PinEntryField.kt

```diff
@@ -31,10 +29,10 @@ import androidx.compose.ui.text.input.TextFieldValue
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import de.digitalService.useID.R
-import de.digitalService.useID.ui.theme.UseIDTheme
+import de.digitalService.useID.ui.theme.UseIdTheme
 
 @Composable
-fun PINEntryField(
+fun PinEntryField(
```

Comment:
The aligment of the digits in the PIN entry field seems a little bit odd. Could you please take a look into that one?

Reply:
Improved :)

Reply:
For me the entry field still looks a little different from the design. Could you please check again with the design? What differs in my eyes:
- Transport-PIN: Digits should be aligned horizontally equally.
- The entry fields should have rounded corners.
- The dots in the redacted mode should be a little larger.

Reply:
You are absolutely right, sorry that I didn't look more carefully 😅  let me know if its okay now :)

Reply:
Nailed it. 😉

Reply:
Oh no, I just saw that the PIN entry field in the identification flow is stilly incorrect. 🫣

Reply:
Ohh I am sorry :( Now all the pin entries should look okay. I hope I didn't miss one again 😅

Reply:
Looks good. ;)